Emby.Server.Implementations/ScheduledTasks/Tasks/AudioNormalizationTask.cs

```diff
@@ -76,6 +76,10 @@ public AudioNormalizationTask(
     /// <inheritdoc />
     public async Task ExecuteAsync(IProgress<double> progress, CancellationToken cancellationToken)
     {
+        var numComplete = 0;
+        var numlibraries = _libraryManager.RootFolder.Children.Count();
```

Comment:
This won't work if not all libraries are Music libraries and/or not all music libraries have enabled LUFS scan. You have to either always report the progress later(in line 88) when we breaks the loop early, or only count the libraries that have enabled library scan.